go.work.sum

```diff
@@ -389,10 +389,16 @@ contrib.go.opencensus.io/exporter/prometheus v0.4.1 h1:oObVeKo2NxpdF/fIfrPsNj6K0
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9 h1:VpgP7xuJadIUuKccphEpTJnWhS2jkQyMt6Y7pJCD7fY=
 dubbo.apache.org/dubbo-go/v3 v3.0.4/go.mod h1:GVCUBZwVkqFJFCtSgIUS7APbv2RBT4KB7sNuLK7D+ko=
 entgo.io/ent v0.9.0 h1:2S1zfpMMW6p+wctj6kcYUprNPNjLWFW06T5MdyAfmWc=
+github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 h1:/vQbFIOMbk2FiG/kXiLl8BRyzTWDw7gX/Hz7Dd5eDMs=
```

Comment:
Why do we need to update this file? It looks like not related to this PR. Please restore it.